src/main/resources/config/liquibase/changelog/20230122022904_changelog.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+    <changeSet author="baboci" id="20230122022904-1">
+        <addColumn tableName="exam_user">
+            <column name="created_by" type="varchar(45)"/>
+        </addColumn>
+        <addColumn tableName="exam_user">
+            <column name="last_modified_by" type="varchar(45)"/>
+        </addColumn>
+        <addColumn tableName="exam_user">
+            <column name="last_modified_date" type="timestamp"/>
+        </addColumn>
+        <addColumn tableName="exam_user">
+            <column name="created_date" type="timestamp"/>
+        </addColumn>
+        <dropForeignKeyConstraint baseTableName="exam_user" constraintName="FK9kic2piykbv64dc8kjcxnlagu" />
+        <dropForeignKeyConstraint baseTableName="exam_user" constraintName="FKpvet665enmcbtnoecju2epqmp" />
+
+        <dropPrimaryKey tableName="exam_user" />
+        <addColumn tableName="exam_user">
```

Comment:
we have existing rows in the table. I don't think this will work. At least it would be unclear which ids the existing columns would get.

We need to think about a migration

Reply:
I just tried it out and it seems to work. So in case liquibase generates something like the following, it should be fine.

```
ALTER TABLE `ArTEMiS`.`exam_user_clone`
ADD COLUMN `id` bigint NOT NULL auto_increment,
ADD PRIMARY KEY (`id`);
```